docs/blog/posts/buckets_as_conceptual_hydr.md

```diff
@@ -0,0 +1,103 @@
+
+But let's take this idea of buckets flowing into each other a bit more seriously as a representation of hydrologic processes. When I am out on Carmel Valley Road overlooking the watershed, I see depressions, gullies, rills, blades of grass, everything else. All these individual processes do somewhat behave as oddly shaped buckets. A blad of grass, for instance, does collect moisture from the air, and that moisture does flow down the blade. A prarie pothole also fills up and overflows, bucket-like, yet some water flows down through the leaking bottom. Though these are not perfectly round reservoirs with known leaks with known coefficients, they do all sort of flow into each other. I used this idea to try to bring neural networks into hydrology from a more realistic conceptualization. With differentiable modeling (Shen et al., 2023), we can set up our neural network to behave like these systems of buckets, complete with valves on the spigots determining the flow coefficient (weights), and the water level dropping below the spigot shutting off flow completely (activation). What I think we are left with here is a pretty darn good digital interpretation of a messy, complex, heterogeneous system (image below from Frame et al., 2024).  
+
+[![Method Discussion](https://github.com/jmframe/nash_cascade_neural_network/blob/main/poster/method_discussion.png?raw=true)](https://github.com/jmframe/nash_cascade_neural_network/blob/main/poster/method_discussion.png?raw=true)
+
+Each individual bucket in our nash cascade neural network behaves like a mass concerving perceptron (Wang et al., 2024). In this scenario, we end up with a model that has the uncanney ability to match a downstream diagnostic variable very well, and we have utilized the hydrologists tool of choice. But if we look closely, are we not simply re-creating a neural network?  
+
+We've seen overwhelming evidence in the past few decades that neural networks are more accurate at predicting hydrologic responses than the conceptualization of hydrologic systems as buckets (Nearing et al., 2020). Unless that conceptualization itself can be boosted by a neural network (Shen et al., 2024). What I've always pondered is what would happen if hydrologists acted a bit more like physicists, and rejected demonstratively predictive conceptualizations in favor of those that match observations. There is often a sentiment that the bucket conceptualizations, even as simplifications, are "interpretable", but an interpretation is only usefull if we are honest about them limitations. We still learn the Rayleigh–Jeans law in thermodynamics, but we then understand the limitations of the continuous energy conceptualization and turn to Planck's Law to grasp the quantum reality. We can still learn bucket conceptualizations, but let's not pretend that they are "physical" representations.  
```

Comment:
The sentence "Unless that conceptualization itself can be ..." is an incomplete sentence. 

Also, I'm a bit confused "demonstratively predictive conceptualization" refers to NN or bucket model? 

If it refers to bucket model, some word containing a bit more essence of "empirical", "conventional", "what people are familiar with," or "deterministic bucket" would be helpful. 

If it refers to a NN, some terms inferring "simply provides accurate predictions" or "simply a large network of bucket models in the end" would be helpful.

Reply:
Ahh, this: "demonstratively predictive conceptualization" should be "demonstratively poor...". 

It is any model conceptualization.

Re-phrased a bit, I think it should be more clear.

Reply:
I see, then the sentence makes sense now !! 👍🏻